opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/package/_source/conf.py.in

```diff
@@ -18,7 +18,7 @@ sys.path.insert(0, os.path.abspath('../opentera/webrtc/native_client'))
 
 for path in sys.path:
     for prefix in "${PYTHON_HTML_IGNORE_PREFIXES}".split(";"):
-        if path.startswith(prefix):
+        if prefix.strip() != "" and path.startswith(prefix):
```

Comment:
Don't remove from path if prefix is empty, because in that case, we clear the whole path as `startswith("")` is always True